pkg/validate/security_context.go

```diff
@@ -46,7 +49,7 @@ var _ = framework.KubeDescribe("Security Context", func() {
 		ic = f.CRIClient.CRIImageClient
 	})
 
-	Context("runtime should support podSandbox with security context", func() {
+	Context("runtime should support NamespaceOption", func() {
```

Comment:
because #19 is closed, so I rename it from `podSandbox with security context` to `NamespaceOption`.